src/db.rs

```diff
@@ -170,6 +170,12 @@ impl Database {
         // parse
         let stmts = parse(sql)?;
 
+        if sql.to_lowercase().starts_with("show ") {
```

Comment:
I would suggest returning error in the binder level, instead of simply doing string matching. What if user input is `<space>show tables`?

You may add a new match arm at 

https://github.com/risinglightdb/risinglight/blob/de55dfe2b53e9d6d623e330d5f15d40e721cfa8a/src/binder/mod.rs#L110-L122

And return a special error (like SQL not supported) for `show` statements.